### ISSUE TITLE
Wrong Namespace

### ISSUE THREAD

**Issue by mcNETDev** [nightowlengineer/sonos-api-java#16]
**Describe what's going wrong**
Just found a wrong namespace 
```java
Exception in thread "main" SonosGlobalError{errorCode=ERROR_UNSUPPORTED_NAMESPACE, reason='Namespace not found'}
at engineer.nightowl.sonos.api.resource.GroupResource.modifyGroupMembers(GroupResource.java:105)

String.format("/v1/groups/%s/modifyGroupMembers", groupId)
must be
String.format("/v1/groups/%s/groups/modifyGroupMembers", groupId)

same with setGroupMembers
```
described in `https://developer.sonos.com/reference/control-api/groups/modifygroupmembers/`

**Environment:**
 - sonos-api-java version: [e.g. 0.0.22]
 - Sonos API version: [e.g. v1]

**Comment by nightowlengineer** [comment]
Thanks for reporting - I'll try and fix this shortly 🙂

**Comment by nightowlengineer** [comment]
Fixed in 0.0.23 which is being released now. I did spot a couple of other issues which I'll get fixed soon, but this fixes the immediate path issue.

**Comment by nightowlengineer** [comment]
Closing this off - let me know if you have any issues with the latest versions, I've just released 0.0.24 with some additional fixes and dependency bumps in.
